packages/create-expo-app/jest.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+
+const roots = ['./src'];
```

Comment:
I'd recommend splitting this into two different scripts like we do with expo/cli and have one set of testing be part of the check packages, and the other be part of an e2e test workflow like we do with CLI and fingerprint.